### ISSUE TITLE
lein daemon start :name-of-service failed

### ISSUE THREAD

**Issue by hozumi** [arohner/lein-daemon#9]
https://github.com/arohner/lein-daemon/blob/master/src/leiningen/daemon_starter.clj#L12
The value of alias in the above line is not keyword, but string.
So retrieving daemon map always fail.

```
% cat :http.log
java.lang.NullPointerException
    at clojure.lang.Symbol.intern(Symbol.java:60)
    at clojure.core$symbol.invoke(core.clj:523)
    at leiningen.daemon_starter$daemon_starter.doInvoke(daemon_starter.clj:13)
    at clojure.lang.RestFn.invoke(RestFn.java:423)
    at clojure.lang.Var.invoke(Var.java:419)
    at clojure.lang.AFn.applyToHelper(AFn.java:163)
    at clojure.lang.Var.applyTo(Var.java:532)
    at clojure.core$apply.invoke(core.clj:603)
    at leiningen.core.main$resolve_task$fn__1606.doInvoke(main.clj:132)
    at clojure.lang.RestFn.applyTo(RestFn.java:139)
    at clojure.lang.AFunction$1.doInvoke(AFunction.java:29)
    at clojure.lang.RestFn.applyTo(RestFn.java:137)
    at clojure.core$apply.invoke(core.clj:603)
    at leiningen.core.main$apply_task.invoke(main.clj:167)
    at leiningen.core.main$_main$fn__1665.invoke(main.clj:237)
    at leiningen.core.main$_main.doInvoke(main.clj:221)
    at clojure.lang.RestFn.invoke(RestFn.java:421)
    at clojure.lang.Var.invoke(Var.java:419)
    at clojure.lang.AFn.applyToHelper(AFn.java:163)
    at clojure.lang.Var.applyTo(Var.java:532)
    at clojure.core$apply.invoke(core.clj:601)
    at clojure.main$main_opt.invoke(main.clj:324)
    at clojure.main$main.doInvoke(main.clj:427)
    at clojure.lang.RestFn.invoke(RestFn.java:457)
    at clojure.lang.Var.invoke(Var.java:427)
    at clojure.lang.AFn.applyToHelper(AFn.java:172)
    at clojure.lang.Var.applyTo(Var.java:532)
    at clojure.main.main(main.java:37)
```

**Comment by arohner** [comment]
For now, lein daemon only does a simple read-string, and "lein daemon start foo" foo is read as a string, but "lein daemon start :foo", foo is read as a keyword. That means your project.clj will need to contain :daemon {:foo or :daemon {"foo" as appropriate.

**Comment by hozumi** [comment]
Sorry for my lack of explanation.
I think that you may misunderstood this problem.

`project.clj`

```
:daemon {:name-of-service {:ns my.name.space
                           :pidfile "path-to-pidfile.pid"}}
```

```
lein daemon start :name-of-service
```

The above combination, which is described in README.md, don't work, because :name-of-service in the `lein daemon-starter :name-of-service`[1] isn't converted to keyword [2].

[1] https://github.com/arohner/lein-daemon/blob/master/src/leiningen/daemon.clj#L62
[2] https://github.com/arohner/lein-daemon/blob/master/src/leiningen/daemon_starter.clj#L12

To avoid this problem, users must specify two almost same entry as following.

```
:daemon {:name-of-service {:ns my.name.space
                           :pidfile "path-to-pidfile.pid"}
        ":name-of-service" {:ns my.name.space
                           :pidfile "path-to-pidfile.pid"}}
```

I think this behavior is not spec, but bug.
Thanks.

**Comment by seancorfield** [comment]
See #10 - I also ran into this and agree with @hozumi that the documentation and behavior do not agree.

**Comment by arohner** [comment]
Fixed in 0.5.3

**Comment by arohner** [comment]
Strings and keywords are now interchangeable in the daemon map :daemon {:foo} and :daemon {"foo"} both work. At the command line, "lein daemon foo" should always be used.

**Comment by hozumi** [comment]
Allen, Thank you.

**Comment by jeluard** [comment]
Main README.md still refers to using : syntax ("lein daemon start :name-of-service"). Also version defined is 0.5.1.
Would be safer to update those!

Thanks.

**Comment by jeluard** [comment]
Great! Actually there is still one wrong usage of `:` in the `README.md` in the first paragraph :)

**Comment by jeluard** [comment]
Also console output you get when running `lein daemon` refers to the old syntax.

edit: this one as actually been fixed in #16.
